applications/nrf5340_audio/src/bluetooth/bt_management/bt_mgmt.h

```diff
@@ -123,7 +123,14 @@ int bt_mgmt_scan_start(uint16_t scan_intvl, uint16_t scan_win, enum bt_mgmt_scan
 int bt_mgmt_manufacturer_uuid_populate(struct net_buf_simple *uuid_buf, uint16_t company_id);
 
 /**
- * @brief	Create and start advertising for ACL connection.
+ * @brief	Stop advertising for an ACL connection.
```

Comment:
Might not related to this PR, but we could remove ACL, since ADV might not only for ACL connection, but just a BLE ADV like broadcasting.